pennylane/io/qualtran_io.py

```diff
@@ -28,6 +29,94 @@
     pass
 
 
+# pylint: disable=unused-argument
+@lru_cache
+def _get_to_pl_op():
+
+    @singledispatch
+    def _to_pl_op(bloq, wires):
+        return FromBloq(bloq=bloq, wires=wires)
+
+    @_to_pl_op.register
+    def _cnot(bloq: qt.bloqs.basic_gates.CNOT, wires):
+        return qml.CNOT(wires=wires)
+
+    @_to_pl_op.register
+    def _(bloq: qt.bloqs.basic_gates.GlobalPhase, wires):
+        return qml.GlobalPhase(bloq.exponent * np.pi, wires)
```

Comment:
Why `* np.pi` is needed?

Reply:
There is a difference of pi in the implementation for GlobalPhase between Qualtran and PL, this just takes care of that. [See this discussion](https://github.com/quantumlib/Qualtran/pull/1559#pullrequestreview-2683039390) for more details.